source/index.html

```diff
@@ -113,46 +116,47 @@ <h2 class="example__title">Живой пример</h2>
+              <svg width="16" height="16" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M7.98.06c1.34 0 2.43.02 3.27.05a6 6 0 0 1 1.92.37 3.99 3.99 0 0 1 2.31 2.32 6 6 0 0 1 .38 1.92 91.88 91.88 0 0 1 0 6.53 6 6 0 0 1-.38 1.92 3.99 3.99 0 0 1-2.3 2.31 6 6 0 0 1-1.93.38 91.88 91.88 0 0 1-6.53 0 6 6 0 0 1-1.92-.38 3.99 3.99 0 0 1-2.32-2.3 6 6 0 0 1-.37-1.93 91.73 91.73 0 0 1 0-6.53A6 6 0 0 1 .48 2.8 3.99 3.99 0 0 1 2.8.48 6 6 0 0 1 4.72.11C5.56.08 6.65.06 7.98.06Zm0 1.03c-1.38 0-2.51.02-3.39.05-.56.02-1.08.12-1.56.3-.75.31-1.28.84-1.6 1.6-.17.47-.27 1-.29 1.55a98.83 98.83 0 0 0 0 6.79c.02.56.12 1.08.3 1.56.31.75.84 1.28 1.6 1.6.47.17 1 .27 1.55.29a98.66 98.66 0 0 0 6.79 0 5.01 5.01 0 0 0 1.56-.3 2.85 2.85 0 0 0 1.6-1.6c.17-.47.27-1 .29-1.55a98.66 98.66 0 0 0 0-6.79 5.01 5.01 0 0 0-.3-1.56 2.85 2.85 0 0 0-1.6-1.6c-.47-.17-1-.27-1.55-.29-.88-.03-2.01-.05-3.4-.05Zm0 10.07c.88 0 1.63-.31 2.25-.93.62-.62.93-1.37.93-2.25 0-.87-.31-1.62-.93-2.24a3.06 3.06 0 0 0-2.25-.93c-.87 0-1.62.31-2.24.93a3.06 3.06 0 0 0-.93 2.24c0 .88.31 1.63.93 2.25.62.62 1.37.93 2.24.93Zm0-7.24c1.13 0 2.09.4 2.88 1.19.8.8 1.19 1.75 1.19 2.87 0 1.13-.4 2.09-1.2 2.88-.78.8-1.74 1.19-2.87 1.19-1.12 0-2.08-.4-2.87-1.2a3.92 3.92 0 0 1-1.19-2.87c0-1.12.4-2.08 1.19-2.87.8-.8 1.75-1.19 2.87-1.19Zm5.2-.15a.95.95 0 0 1-.97.94.9.9 0 0 1-.67-.28.9.9 0 0 1-.27-.66.95.95 0 0 1 .93-.97c.26 0 .49.1.68.29.2.19.3.42.3.68Z" fill="#!666"/></svg>
               <span class="visually-hidden">мы в instagram</span>
             </a>
           </li>
           <li class="social__item">
             <a class="social__link" href="#!">
-              <svg width="22" height="22" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M16.5 1.83h-2.75c-1.22 0-2.38.48-3.24 1.32a4.45 4.45 0 0 0-1.34 3.18v2.7H6.42v3.59h2.75v7.19h3.66v-7.2h2.75l.92-3.59h-3.67v-2.7c0-.23.1-.46.27-.63.17-.17.4-.26.65-.26h2.75v-3.6Z" stroke="#666" stroke-width="1.1" stroke-linecap="round" stroke-linejoin="round"/></svg>
+              <svg width="22" height="22" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M16.5 1.83h-2.75c-1.22 0-2.38.48-3.24 1.32a4.45 4.45 0 0 0-1.34 3.18v2.7H6.42v3.59h2.75v7.19h3.66v-7.2h2.75l.92-3.59h-3.67v-2.7c0-.23.1-.46.27-.63.17-.17.4-.26.65-.26h2.75v-3.6Z" stroke="#!666" stroke-width="1.1" stroke-linecap="round" stroke-linejoin="round"/></svg>
```

Comment:
можно убрать лишние атрибуты в разметке
```suggestion
              <svg width="22" height="22" fill="none"><path d="M16.5 1.83h-2.75c-1.22 0-2.38.48-3.24 1.32a4.45 4.45 0 0 0-1.34 3.18v2.7H6.42v3.59h2.75v7.19h3.66v-7.2h2.75l.92-3.59h-3.67v-2.7c0-.23.1-.46.27-.63.17-.17.4-.26.65-.26h2.75v-3.6Z" stroke="#!666" stroke-width="1.1" stroke-linecap="round" stroke-linejoin="round"/></svg>
```